templates/common/_base/files/mtu-migration-conf.yaml

```diff
@@ -0,0 +1,22 @@
+{{ $target_mtu := 0 -}}
```

Comment:
I'm not sure I understand why this logic is here. Shouldn't these variables be set somewhere else? And then you can just make use of them in the contents?

Reply:
I am making use of `.MTUMigration.Network.To` and `.MTUMigration.Network.From`. This is just go template syntactic sugar.

Reply:
Please note that there is no short circuit of conditionals in go template yet and we can't simply do `and .MTUMigration.Network .MTUMigration.Network.To`. It can only be done by embedding if's so it is written like this for easy readibility.